linux/utils/shutdown.yml

```diff
@@ -9,6 +9,17 @@
   changed_when: false
   async: 1
   poll: 0
+  when: guest_os_family != "FreeBSD"
+
+# The 'shutdown -h now' will let FreeBSD to halt
+- name: Execute guest OS shutdown for FreeBSD
+  ansible.builtin.command: "poweroff"
```

Comment:
You can add a variable like `guest_shutdown_cmd` at the beginning of this task file, and set command for different OS family:
```
guest_shutdown_cmd: "{{ 'poweroff' if guest_os_family == 'FreeBSD else 'shutdown -h now' }}"
```

Reply:
Change it. Thanks for your suggestion.